src/Components/Trending/Trending.jsx

```diff
@@ -0,0 +1,65 @@
+import "./Trending.scss";
+
+const Trending = () => {
+  return (
+    <>
+      <div className="head">
+        <h1>GUIDANCE GRID</h1>
+      </div>
+      <div className="searchbar">
+        <input type="text" placeholder="Search blogs" />
+      </div>
+      <secttion className="trending">
+        <h3>Trending Articles</h3>
+        <div className="trend-art1">
```

Comment:
Instead of limiting the card component for a row and styling the row. Use `flex` or `grid` instead. Because you don't know how many `TrendingCards` will be there. 

```html
<div className="trend-art1">
    <TrendingCard id={id} profileImg={profileImg} title={title} description={description} time={time} />
    <TrendingCard id={id} profileImg={profileImg} title={title} description={description} time={time} />
    <TrendingCard id={id} profileImg={profileImg} title={title} description={description} time={time} />
</div>
<div className="trend-art2">
    <TrendingCard id={id} profileImg={profileImg} title={title} description={description} time={time} />
    <TrendingCard id={id} profileImg={profileImg} title={title} description={description} time={time} />
    <TrendingCard id={id} profileImg={profileImg} title={title} description={description} time={time} />
</div>
<div className="trend-art3">
    <TrendingCard id={id} profileImg={profileImg} title={title} description={description} time={time} />
    <TrendingCard id={id} profileImg={profileImg} title={title} description={description} time={time} />
    <TrendingCard id={id} profileImg={profileImg} title={title} description={description} time={time} />
</div>
```

Instead display as many `TrendingCard` component you want using loop

```html
<div className={style["trending-card-container"]}>
    {trending.map(t => (
        <TrendingCard key={t.id} id={t.id} profileImg={t.profileImg} title={t.title} description={t.description} time={t.time} />
    ))}
</div>
```

And style the parent class (here `trending-card-container`) using `flex` or `grid` properties.